providers/catalog.go

```diff
@@ -18,6 +18,8 @@ const (
 	Dropbox    Provider = "dropbox"
 	Notion     Provider = "notion"
 	Gong       Provider = "gong"
+	DocuSign          Provider = "docuSign"
```

Comment:
Still seems like there is an indentation issue here. Please run `make fix` in the repo to fix it

Reply:
![Screenshot 2024-03-19 at 12 41 53 PM](https://github.com/amp-labs/connectors/assets/18614743/6a4d5fdb-e2df-4c9d-991e-a2d503d09a16)